airflow/providers/apache/druid/README.md

```diff
@@ -0,0 +1,123 @@
+| [apache-airflow-backport-providers-apache-hive](https://github.com/apache/airflow/tree/master/airflow/providers/apache/hive) | apache.hive |
+
+# Provider class summary
+
+All classes in Airflow 2.0 are in `airflow.providers.apache.druid` package.
+
+
+## Operators
+
+
+
+
+### Moved operators
+
+| Airflow 2.0 operators: `airflow.providers.apache.druid` package                                                                                        | Airflow 1.10.* previous location (usually `airflow.contrib`)                                                                                               |
+|:-------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [operators.druid.DruidOperator](https://github.com/apache/airflow/blob/master/airflow/providers/apache/druid/operators/druid.py)                       | [operators.druid_operator.DruidOperator](https://github.com/apache/airflow/blob/v1-10-stable/airflow/contrib/operators/druid_operator.py)                  |
+| [operators.druid_check.DruidCheckOperator](https://github.com/apache/airflow/blob/master/airflow/providers/apache/druid/operators/druid_check.py)      | [airflow.operators.druid_check_operator.DruidCheckOperator](https://github.com/apache/airflow/blob/v1-10-stable/airflow/operators/druid_check_operator.py) |
+| [operators.hive_to_druid.HiveToDruidTransfer](https://github.com/apache/airflow/blob/master/airflow/providers/apache/druid/operators/hive_to_druid.py) | [airflow.operators.hive_to_druid.HiveToDruidTransfer](https://github.com/apache/airflow/blob/v1-10-stable/airflow/operators/hive_to_druid.py)              |
+
+
+
+
+
+## Hooks
+
+
+
+### Moved hooks
+
+| Airflow 2.0 hooks: `airflow.providers.apache.druid` package                                                               | Airflow 1.10.* previous location (usually `airflow.contrib`)                                                               |
+|:--------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------|
+| [hooks.druid.DruidDbApiHook](https://github.com/apache/airflow/blob/master/airflow/providers/apache/druid/hooks/druid.py) | [airflow.hooks.druid_hook.DruidDbApiHook](https://github.com/apache/airflow/blob/v1-10-stable/airflow/hooks/druid_hook.py) |
+| [hooks.druid.DruidHook](https://github.com/apache/airflow/blob/master/airflow/providers/apache/druid/hooks/druid.py)      | [airflow.hooks.druid_hook.DruidHook](https://github.com/apache/airflow/blob/v1-10-stable/airflow/hooks/druid_hook.py)      |
+
+
+
+
+
+
+## Releases
+
+### Release 2020.05.11
```

Comment:
```suggestion
### Release 2020.05.11
```

We will need to add 3 days for vote, so if we are planning to start the vote today and release RCs today, this should be `2020.05.14`

Reply:
I can easily change it when we make a real RC (as I did now), but I think the date should be att the time of tag rather than time of actual release (and we should not change it after the voting - we should use the same sources/tag to build the RC and final version. This means that release will always be released a few days later than the date. But I think it's perfectly fine. 

Eventually - even if takes few days to vote, the tag should not move and it will be a snapshot as of the date of voting. So for release I think I will merge it with the YYYY.MM.DD date, but to build the package. I am thinking now that I should add a feature to change the version via parameter when building the package - so that I can build the rc version from the same sources.

Reply:
WDYT ?

Reply:
That will become confusing, the release tag would contain the date and should correspond to the actual version too. That is what we follow with Airflow's Changelog too. We add a date with RC (+3 or + whatever days we open the vote for).

If the vote is not passed the reason because we need to fix something, so we change the date too at that time.

I am happy to keep it as it is for now but when we cut RCs let's add +3 days

Reply:
I am generally OK with that, but just one more try -  I think it might get a bit confusing if we have commits added during the vote. 

If that happens, you will have some changes with date before the release, only appearing in the next one, which might confuse people.

We will have it in the release notes though and commit date is not really something you often look at, so I might go for it.

Reply:
Yeah, commits in Master branch shouldn't be looked at anyway :) Users should be looking at a certain tag if they are looking for release notes, changelog etc.

Reply:
Right :), Will do +3 tomorrow. :)

Reply:
Rssolved.